designs/administrator/4.6/guided-tours/index.md

```diff
@@ -0,0 +1,102 @@
+
+![3](img/tour1-screen0.png)
+
+## Tour permissions
+For 4.6, tour permissions will be as follows:
+* If a user can install Operators, show the first Serverless tour (Explore Serverless) which walks users through installing the Serverless Operator.
+* If a user cannot install Operators, do not show the first Serverless tour. Instead show an empty state. The empty state can inform users that tours will be added in future.
+* If an Operator has been installed, show the second Serverless tour (Serverless Applications) which walks users through deploying a Serverless application.
+* Any tour that has been completed should remain on the Guided Tours page with the option to **Review the tour** or **Restart the tour**.
+
+## Tour 1: Explore Serverless
+The first tour walks users through installing the Serverless Operator by completing a series of 3 tasks. Each task will have a number of sub steps that the user should execute in order to successfully complete the tour. Any links within the tour instructions will redirect users to that page in the console.
+
+### Check your work
+There will not be any validation from the console that users have successfully completed any sub steps or tasks. Instead, there will be an optional **Check your work** feature that asks users to validate that the tasks have been completed.
+
+![4](img/check1.png)
+
+The **Check your work** feature will appear after a user clicks **Next**. The user may click **Next** again to skip the validation step, or can read the instructions to make sure they are following the tour correctly.
+
+![5](img/check2.png)
+
+If the user selects **Yes**, the Check your work alert will turn green.
+
+![6](img/check3.png)
+
+Users can click Next again to advance to the next screen, where they will see that the first task has been marked as completed and the second task is shown.
+
+![7](img/tour1-screen6.png)
+
+If the user selects **No**, the alert will turn red and will recommend that they walk back through the steps in order to complete the task.
+
+![8](img/check4.png)
+
+If the user chooses to continue to the next screen anyway, the task will be marked red and will remind the user to go back to the task to check their work.
+
+![9](img/check5.png)
+
+Users can click the numbered task headers to jump between tasks.
+
+### Panel overflow
+In cases where content does not fit in the panel, the content should scroll up and a shadow should be used to indicate that scrolling is possible.
+
+![10](img/tour1-screen17.png)
+
+### Completing a tour
+The final screen of the tour will show users the completed tasks and give them the option to go back or go to the Guided Tours page. In the case of this tour, there will also be a link to start the second tour. The link should redirect users to the Guided Tours page with the tour panel open and loaded to the second tour's introductory screen.
+
+![11](img/tour1-screen18.png)
```

Comment:
I did have a Finish button at some point but it seemed silly since it would do the same thing as using the 'x' in the corner and added another button. In one version there were like 3 or 4 buttons on this final screen so I thought Back and View all tours seemed sufficient. I do share the concern though that there's no official "Finish" here... I wonder whether we should add a primary Finish button and have that take users to a final success screen with some sort of big check icon and the option to start tour 2 or go to tours? Might be weird though if a user hasn't actually finished the tour and since we can't validate we tell them congratulations even if they haven't done anything 😄

Reply:
I see yeah, a final "Success! (we assume)" screen probably isn't needed. 😄

I could go either way on having what could probably be a "Close" primary button in the footer in addition to the X in the top-right. It does duplicate an action that's available elsewhere, but with these tours being somewhat wizard-like I wonder if folks would expect to be able to exit/close/finish/leave this tour within the same region that they've been interacting with to go forward/back so far. Curious about others' thoughts on this as well!

Reply:
I think it could use a little bit more of a definitive finish. We could also do something as simple as add some copy in this final screen (above the Back and View all tours buttons) that says "Congratulations, you have finished this tour...." Which then begs the question if you show a finish message if they did not complete all the steps successfully.

Reply:
Not directly related to this comment thread, but one participant [commented that this was well suited for gamification](https://docs.google.com/spreadsheets/d/1drTmhFEr-aebThe9NaJEtTKoJtkxyMjgrIGU58_ciKs/edit#gid=0&range=A237) so I do think that some of these like "success" moments could be highlighted. Just a thought.